src/test/mir-opt/unusual-item-types.rs

```diff
@@ -1,86 +1,27 @@
 // Test that we don't ICE when trying to dump MIR for unusual item types and
 // that we don't create filenames containing `<` and `>`
-// ignore-tidy-linelength
 
 struct A;
 
+// EMIT_MIR rustc.{{impl}}-ASSOCIATED_CONSTANT.mir_map.0.mir
 impl A {
     const ASSOCIATED_CONSTANT: i32 = 2;
 }
 
 // See #59021
+// EMIT_MIR rustc.Test-X-{{constructor}}.mir_map.0.mir
 enum Test {
     X(usize),
     Y { a: usize },
 }
 
+// EMIT_MIR rustc.E-V-{{constant}}.mir_map.0.mir
 enum E {
     V = 5,
 }
 
 fn main() {
     let f = Test::X as fn(usize) -> Test;
+    // EMIT_MIR rustc.ptr-drop_in_place.std__vec__Vec_i32_.AddMovesForPackedDrops.before.mir
```

Comment:
there was no mir dump emitted for this comment. I think `--bless` only detects comments at the start of the line... not indented ones. I should fix that XD

Reply:
Please just unindent it for now

Reply:
Please also check whether I missed other instances of this